app/POROs/movie.rb

```diff
@@ -0,0 +1,11 @@
+class Movie
+  attr_reader :id,
+              :title,
+              :vote_average
+              
+  def initialize(data)
+    @id = data[:id]
+    @title = data[:title]
+    @vote_average = data[:vote_average]
+  end
```

Comment:
Encapsulation does apply here since we are having the movie poro be responsible for pulling out the necessary information. I would also so that Abstraction is another concept that is applied because the class that passes the poro the data doesn't need to know what information is being pulled out or how this poro uses the data.